be/src/util/bit_stream_utils.h

```diff
@@ -77,6 +77,9 @@ class BitWriter {
     // to the next byte boundary.
     void Flush(bool align = false);
 
+    // Maximum byte length of a vlq encoded int
+    static const int MAX_VLQ_BYTE_LEN = 5;
```

Comment:
why add one new `MAX_VLQ_BYTE_LEN`

Reply:
In the original code, it is local static variables within another class, i can not use it in current class.